src/app/application/components/route-view/RouteView.vue

```diff
@@ -19,7 +21,7 @@
       <DataSink
         v-if="me"
         v-slot="{ submit }"
-        :src="`/me/mesh-list-view`"
+        :src="`/me/${props.name}`"
```

Comment:
This probably should have been done in https://github.com/kumahq/kuma-gui/pull/2625, but seeing as https://github.com/kumahq/kuma-gui/pull/2625 was just doing the `mesh-list` anyway it doesn't really matter.

This change means we take the `<RouteView name="this-bit">` and use that as a unique key to save the data against in localStorage. This means every route has its own namespace to save things in.